src/index.ts

```diff
@@ -124,31 +130,76 @@ function fileToText(file: File): Promise<string> {
   });
 }
 
-class MultinetAPI {
-  public client: Client;
+function extract<T>(promise: AxiosPromise<T>): Promise<T> {
+  return new Promise((resolve, reject) => {
+    promise.then((resp) => resolve(resp.data))
+      .catch((err) => reject(err));
+  });
+}
```

Comment:
Is the whole point of the above function just to extract the `.data` property from an object? Seems like that may not be necessary, and you could just destructure the result i.e. `const { data } = await ...`

Reply:
Yeah, I made the choice to do some extra work in crafting this library to save a small amount of cognitive load on application writers. Perhaps it's the wrong choice. Check out [this discussion](https://github.com/multinet-app/multinetjs/pull/30#discussion_r597148096) where Jeff brought up the same comment.

Reply:
I'd vote for removing this and just embracing our use of the axios API. Maybe a downstream will actually want to look at the HTTP status code of the response, or one of the headers. Dan Lamanna has convinced me of the virtue of avoiding "wrapping" a third party library with layers of abstraction that at best provide the same exact amount of power (but usually not as much) and also creating a lot of boilerplate code. To me, it's not a hard sell in terms of cognitive load: we just get to say that our API is essentially a superset of the axios API, which is already very popular, powerful, and well designed and documented.

Reply:
Now that you and Jeff have both brought up the exact same concern, I'm starting to think we should just simplify this. The change was actually instituted in that other PR, so I'd do the simplification there and rebase this PR appropriately.

@AlmightyYakob, any objection to taking Jeff and Zach's suggestion to heart?